lib/models/mirror.js

```diff
@@ -3,6 +3,7 @@
 const errors = require('storj-service-error-types');
 const mongoose = require('mongoose');
 const SchemaOptions = require('../options');
+const _ = require('lodash');
```

Comment:
Hmm..lodash was there when I wrote the test which is why I used `_.isEmpty`...guess it must have been put it from another merge somewhere. Test can be rewritten to not use lodash.